### ISSUE TITLE
Output: interpolated strings & culture

### ISSUE THREAD

**Issue by ig-sinicyn** [dotnet/BenchmarkDotNet#307]
There're places in the codebase that fails to pass the host culture into formatted strings,
[as example](https://github.com/dotnet/BenchmarkDotNet/blob/24e944ae06101a8051955fcfb0ae6c4413950fdc/src/BenchmarkDotNet.Core/Extensions/StatisticsExtensions.cs#L37):
```
            builder.AppendLine($"Skewness = {s.Skewness}, Kurtosis = {s.Kurtosis}");
```

the output is
```
ConfidenceInterval = [45.8375 us; 45.8375 us] (CI 95%)
Skewness = не число, Kurtosis = не число
```
**hotfix:** just add `.ToStr()`

Sadly, there's no good solution for it. You can wrap all interpolated strings into
```
using static System.FormattableString;
...
            builder.AppendLine(Invariant($"Skewness = {s.Skewness}, Kurtosis = {s.Kurtosis}"));
```
but it'll result in additional `FormattableString` [instantiation per call](https://github.com/dotnet/roslyn/issues/518), not to mention issue with targeting to 4.5.
Long story short, string interpolation [does not play well](https://github.com/NLog/NLog/issues/825) with logging : (

**Comment by AndreyAkinshin** [comment]
It seems that we should just use the `.ToStr()` hotfix, I don't know a good solution for this problem.
